### PR TITLE
feat: sonar-fix-sprint-9-geo-355

### PR DESCRIPTION
**Pull request by andybcgov**
Sonar fixes sprint 9

https://finrms.atlassian.net/browse/GEO-355

---
Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://pay-transparency-pr-322-frontend.apps.silver.devops.gov.bc.ca)

### CODE REVIEW

File: doc-gen-service/src/templates/report-template-header.html [hunk 1]
```diff
@@ -32,9 +32,9 @@
   }
 
   .pay-transparency-report .watermark .watermark-body {
-    color: #bbbbbb;
+    color: #cccccc;
```
Comment (sukanya-rath) [inline doc-gen-service/src/templates/report-template-header.html:35]: remove this change

File: doc-gen-service/src/templates/report-template-header.html [hunk 2]
```diff
@@ -32,9 +32,9 @@
...
     opacity: 0.15;
-    font-size: 190px;
+    font-size: 100px;
```
Comment (sukanya-rath) [inline doc-gen-service/src/templates/report-template-header.html:37]: remove this change

**Review by sukanya-rath — CHANGES_REQUESTED** [review]

**Review by sukanya-rath — APPROVED** [review]

**Review by sukanya-rath — CHANGES_REQUESTED** [review]
please fix the sonar warning

**Comment by andybcgov** [comment]
> please fix the sonar warning

@sukanya-rath There's an error generating the report. I checked quickly but couldn't find the cause, I'll keep looking. FYI - my CSV is 5KB .
image

**Comment by andybcgov** [comment]
> please fix the sonar warning

@sukanya-rath I think I've fixed this already. Not sure though.

**Review by sukanya-rath — APPROVED** [review]

**Pull request merged**